src/project.cc

```diff
@@ -24,6 +24,10 @@
 #include <unordered_set>
 #include <vector>
 
+#ifdef _WIN32
+  #include <direct.h>
```

Comment:
Add a ChangeWorkingDirectory function to platform.h, with an empty implementation for platform_posix.cc

Reply:
Should I still check for _WIN32 before actually calling ChangeWorkingDirectory? (Seems clearer to me to explicitly show that the directory change is only done on Windows instead of implying it by making ChangeWorkingDirectory a noop on POSIX)

Reply:
You still need to check for _WIN32. It is a hack for clang-cl

Reply:
So I add a ChangeWorkingDirectory function to platform, implement it with _chdir for Windows and chdir for POSIX and then check for _WIN32 in project.cc to make sure ChangeWorkingDirectory is only called on Windows?

Reply:
Yep, sg.

Reply:
No idea, clang-cl accepts the -working-directory header without any warning or error so it might just be a bug that's shared between clang and clang-cl. @MaskRay Do you know if -working-directory doesn't work in clang.exe either?

Reply:
I suspect clang-cl silently drops the flag since it is doing MSVC style parsing, which likely doesn't have an equivalent.

Reply:
I'll try and verify

Reply:
https://msdn.microsoft.com/en-us/library/fwkeyyhe.aspx yea, I don't see anything like -working-dir.